db/connect.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
```

Comment:
Didn't we agree that we didn't need these anymore w/ ES6?

Reply:
I'm not 100% on if we should get rid of it or not, but if we do, we should get rid of all of them at the same time in a separate PR. So for now I just want to be consistent with the rest of the project

Reply:
Sounds good, we can handle that later.